### PR TITLE
feat(utility): migrate repr related source code from TensorBaySDK

### PR DESCRIPTION
**Pull request by wangyuqing0424**

### CODE REVIEW

File: graviti/utility/lazy.py [hunk 1]
```diff
@@ -22,7 +22,8 @@
 )
 
 import pyarrow as pa
-from tensorbay.utility import ReprMixin, ReprType
+
+from graviti.utility import ReprMixin, ReprType
```
Comment (graczhual) [inline graviti/utility/lazy.py:26]: graviti.utility.repr

File: graviti/utility/attr.py [hunk 2]
```diff
@@ -18,7 +18,7 @@
     overload,
 )
 
-from tensorbay.utility import ReprMixin, ReprType
+from graviti.utility import ReprMixin, ReprType
```
Comment (graczhual) [inline graviti/utility/attr.py:21]: graviti.utility.repr

**Review by graczhual — CHANGES_REQUESTED** [review]

**Review by Lee-000 — CHANGES_REQUESTED** [review]
The imports in `graviti/schema/package.py ` should be updated.

**Review by graczhual — APPROVED** [review]

File: graviti/schema/package.py [hunk 3]
```diff
@@ -12,9 +12,10 @@
 from typing import TYPE_CHECKING, Any, Dict, Iterator, List, Mapping, Optional, Tuple, Type, TypeVar
 
 import yaml
-from tensorbay.utility import ReprMixin, ReprType, UserMapping
+from tensorbay.utility import UserMapping
 
-from graviti.utility import AttrDict
+from graviti.utility.attr import AttrDict
+from graviti.utility.repr import ReprMixin, ReprType
```
Comment (Lee-000) [inline graviti/schema/package.py:18]: `from graviti.utility import AttrDict, ReprMixin, ReprType`?
Imports from another module.

Comment (wangyuqing0424) [reply graviti/schema/package.py:18]: But “from graviti.utility import AttrDict, ReprMixin, ReprType” cannot pass Build Docs check

**Review by Lee-000 — APPROVED** [review]

File: graviti/schema/package.py (hunk 3 above)
Comment (linjiX) [reply graviti/schema/package.py:18]: This is not expected, @graczhual please help on this.

Comment (graczhual) [reply graviti/schema/package.py:18]: Because the circular import. Should we accept the current code, or adjust the file structure in another pr?

**Review by linjiX — APPROVED** [review]

File: graviti/schema/package.py (hunk 3 above)
Comment (Lee-000) [inline graviti/schema/package.py:18]: Use module imports

**Review by Lee-000 — CHANGES_REQUESTED** [review]

**Review by Lee-000 — APPROVED** [review]

**Pull request merged**